ark/utils/spatial_analysis_utils.py

```diff
@@ -84,34 +76,49 @@ def get_pos_cell_labels(analysis_type, pheno=None, current_fov_neighborhood_data
     Returns:
         list:
             List of all the positive labels"""
+    if(
+        thresh is None
+        or current_fov_channel_data is None
+        or cell_labels is None or current_marker is None
+    ):
+        raise ValueError("Incorrect arguments passed for analysis type")
```

Comment:
Do we really need all these `None` checks, or are you just using this to find where all the old calls of `get_pos_cell_labels` are?